src/test/resources/restapi/securityconfig_nondefault.json

```diff
@@ -170,6 +170,11 @@
     "do_not_fail_on_forbidden" : false,
     "multi_rolespan_enabled" : true,
     "hosts_resolver_mode" : "ip-only",
-    "do_not_fail_on_forbidden_empty" : false
+    "do_not_fail_on_forbidden_empty" : false,
+    "on_behalf_of": {
+      "enabled": true,
+      "signing_key": "VGhpcyBpcyB0aGUgand0IHNpZ25pbmcga2V5IGZvciBhbiBvbiBiZWhhbGYgb2YgdG9rZW4gYXV0aGVudGljYXRpb24gYmFja2VuZCBmb3IgdGVzdGluZyBvZiBleHRlbnNpb25z",
+      "encryption_key": "ZW5jcnlwdGlvbktleQ=="
+    }
```

Comment:
When reading this configuration it isn't clear what the difference is between these keys, can we change their names to make it clear what this impacts?

Reply:
Moving the discussion result here:
 Lets leave like this for now. As @davidlago suggested, we should double check the naming with the product team, especially for the naming of `encryption_key`, to see if we need to include what are we encrypting in the description, such as "role_encryptioin_key".